opentracing/ext/tags_test.go

```diff
@@ -0,0 +1,35 @@
+package ext_test
+
+import (
+	"testing"
+
+	"github.com/opentracing/api-golang/examples/dapperish"
```

Comment:
really it was just me cheating. I've already added testutils package, and for this test I only need minimal set of functionality from the Source implementation, so we could just copy a DaperishTraceContextSource under test utils